pallets/acurast-receiver/src/lib.rs

```diff
@@ -15,64 +15,70 @@ mod benchmarking;
 
 #[frame_support::pallet]
 pub mod pallet {
-	use crate::traits::*;
-	use frame_support::pallet_prelude::*;
-	use frame_system::pallet_prelude::OriginFor;
+    use crate::traits::*;
+    use frame_support::pallet_prelude::*;
+    use frame_system::pallet_prelude::OriginFor;
+    use sp_std::prelude::*;
 
-	/// Configure the pallet by specifying the parameters and types on which it depends.
-	#[pallet::config]
-	pub trait Config: frame_system::Config {
-		/// Because this pallet emits events, it depends on the runtime's definition of an event.
-		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
-		/// The fulfillment payload.
-		type Payload: Parameter
-			+ Member
-			+ MaxEncodedLen
-			+ Clone
-			+ Into<frame_support::inherent::Vec<u8>>;
-		/// Handler to notify the runtime when a new fulfillment is received.
-		type OnFulfillment: OnFulfillment<Self>;
-		/// Handle origin validation
-		type Barrier: ParachainBarrier<Self>;
-	}
+    /// Configure the pallet by specifying the parameters and types on which it depends.
+    #[pallet::config]
+    pub trait Config: frame_system::Config {
+        /// Because this pallet emits events, it depends on the runtime's definition of an event.
+        type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+        /// The fulfillment payload.
+        type Payload: Parameter + Member + Clone + Into<Vec<u8>>;
```

Comment:
`MaxEncodedLen` is important here to force the config type to have a fixed length, right?